endoreg_db/serializers/video/video_metadata.py

```
@@ -0,0 +1,105 @@
+        Raises:
+            ValidationError: If value is not valid JSON or not an array
+        """
+        if not value:
+            return value
+        
+        try:
+            parsed = json.loads(value)
+            if not isinstance(parsed, list):
+                raise serializers.ValidationError(
+                    "sensitive_frame_ids must be a JSON array"
+                )
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)